setup.py

```diff
@@ -92,7 +92,7 @@ def build_js_files(self):
     "jinja2 >= 2.9",
     "typing_extensions; python_version < '3.8.0'",
     "rich >= 11.2.0",
-    "textual >= 0.41.0",
+    "textual >= 0.48.0",
```

Comment:
We'd need to first drop support for Python 3.7, because modern Textual no longer supports 3.7.

Reply:
It looks like, if we don't require Textual 0.48 but make the binding unconditionally, things work "fine" even on old versions of Textual. Pressing ctrl-z at runtime just leads to a message logged to the Textual logs:
```
[14:11:50] INFO                                                                                            app.py:2794
<action> 'suspend_process' has no target. Could not find methods '_action_suspend_process' or 'action_suspend_process'
```

That seems acceptable to me in exchange for letting us continue to support 3.7 for a while longer.